source/wp-content/themes/wporg-news-2021/block-template-parts/header-navigation.html

```diff
@@ -0,0 +1,27 @@
+<!-- wp:group {"align":"wide","className":"site-navigation"} -->
+<div class="wp-block-group alignwide site-navigation">
+	<!-- wp:group {"className":"site-navigation-breadcrumb"} -->
+	<div class="wp-block-group site-navigation-breadcrumb">
+		<!-- wp:navigation {"orientation":"horizontal"} -->
+			<!-- wp:home-link {"label":"News"} /-->
+		<!-- /wp:navigation -->
+		<!-- wp:query-title {"type":"archive","className":"site-navigation-breadcrumb-current"} /-->
+	</div>
+	<!-- /wp:group -->
+
+	<!-- wp:group {"className":"site-navigation-categories"} -->
+	<div class="wp-block-group site-navigation-categories">
+		<!-- wp:html -->
+		<input type="checkbox" id="site-navigation-categories-toggle" class="site-navigation-categories-toggle" aria-label="Open categories list" />
```

Comment:
> Should this be a button instead? Or have a form if it's going to stay an input?

The reason it's a checkbox is so that it can have a toggleable state that will persist when not in focus (unlike hovering over or focusing on the element). I'm not aware of a way to do that with a button.

As far as I can tell, an `input` element [does not](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input#technical_summary) have to have a `form` element parent to be valid HTML.

> or use the same approach as global header menu, for consistency?

Possibly. I didn't think the navigation block would allow for the type of dropdown functionality that the design calls for here, but I could be wrong.

Reply:
Ah, now that I understand this, I actually really like it 👍🏻 

It doesn't seem like a common approach, though, so comments here and/or in the CSS would probably be a kindness to our future selves/others.